src/mpyl/utilities/docker/__init__.py

```diff
@@ -92,3 +92,21 @@ def build(logger: Logger, root_path: str, file_path: str, image_tag: str, target
     stream_docker_logging(logger=logger, generator=logs, task_name=f'Build {file_path}:{target}')
     logger.debug(logs)
     return True
+
+
+def write_env_to_file(project: Project, run_properties: RunProperties) -> str:
```

Comment:
Wondering if this should be here or in `mpyl.project.Properties`

Reply:
You tell me, I have no idea! ;)

Reply:
@koeves I would suggest not to add any logic to `mpyl.project.Properties`, but rather keep it a dumb data container. 
Since `write_env_to_file` is only used in one place, I would suggest to keep it local (to `ephemeraldockerdeploy.py`).

Assuming we need it at all, that is. https://github.com/Vandebron/mpyl/pull/65#discussion_r1149964464